src/constant.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+import {cast} from './cast.js';
+import {Tensor, sizeOfShape} from '../src/lib/tensor.js';
+
+/**
```

Comment:
Please update this function's description and also add descriptions for `start`, `step` and `type` parameters.